I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 13, in script call
    call check_missing_proficiencies from start_6
  File "game/script.rpy", line 181, in script
    $ fix_modifier()
  File "game/script.rpy", line 181, in <module>
    $ fix_modifier()
  File "game/script.rpy", line 66, in fix_modifier
    set_modifier('monthly_budget', Modifier_Obj('Budget', "+", 1000), 'money', collection = 'payroll_monthly')
TypeError: set_modifier() takes 2 positional arguments but 3 positional arguments (and 1 keyword-only argument) were given

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 13, in script call
    call check_missing_proficiencies from start_6
  File "game/script.rpy", line 181, in script
    $ fix_modifier()
  File "D:\RenPy\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\RenPy\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 181, in <module>
    $ fix_modifier()
  File "game/script.rpy", line 66, in fix_modifier
    set_modifier('monthly_budget', Modifier_Obj('Budget', "+", 1000), 'money', collection = 'payroll_monthly')
TypeError: set_modifier() takes 2 positional arguments but 3 positional arguments (and 1 keyword-only argument) were given

Windows-10-10.0.22631 AMD64
Ren'Py 8.1.3.23091805
Mind the School 0.1.4
Mon Aug 12 22:18:00 2024
